docs/UserGuide.md

```diff
@@ -60,24 +66,20 @@ Format: `list`
 
 ### Searching for a food item: `find`
 
-Searches for a food item inside the inventory tracker according to `KEYWORD`, `PRIORITY` or `EXPIRY_DATE`.
+Searches inside the inventory tracker for food items which the descriptions contain any of the given keywords.
```

Comment:
Perhaps can consider "Searches for food items in the inventory with descriptions matching any of the given keywords." instead?

Reply:
I agree with you, thank you!